nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/services/azure/storage/ADLSCredentialsControllerService.java

```diff
@@ -65,16 +64,9 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .build();
 
-    public static final PropertyDescriptor ACCOUNT_KEY = new PropertyDescriptor.Builder()
-            .fromPropertyDescriptor(AzureStorageUtils.ACCOUNT_KEY)
-            .description(AzureStorageUtils.ACCOUNT_KEY_BASE_DESCRIPTION)
-            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
-            .build();
+    public static final PropertyDescriptor ACCOUNT_KEY = AzureStorageUtils.ACCOUNT_KEY;
```

Comment:
Instead of keeping the local `ACCOUNT_KEY` property descriptor, it would be better to replace existing references to use `AzureStorageUtils.ACCOUNT_KEY` directly.

Reply:
I was hesitant to remove this (actually first I removed then decided to keep it this way) because this is a public field so theoretically this could have been used by other (3rd party) processors so removing it would be a breaking change.
But I don't have a strong opinion on this. Would you remove it, @exceptionfactory ?

Reply:
Thanks for following up. Although it is a public field, processor variables are not part of the public API contract, just the property names themselves.  Since the change would still preserve the existing property name, refactoring seems like the best approach.

Reply:
gotcha, updating the PR, thanks